spec.bs

```diff
@@ -1949,12 +1948,14 @@ To <dfn>score and rank a bid</dfn> given an [=auction config=] |auctionConfig|,
 
   The network response has to be parsed to pull out the pieces relevant to each
   [=evaluating a scoring script|evaluation of a scoring script=].
+1. Let |seller| be |auctionConfig|'s [=auction config/seller=].
 1. Let |trustedScoringSignals| be null.
 1. Let «|allTrustedScoringSignals|, ignored, |scoringDataVersion|» be the result of [=fetching
   trusted signals=] with |fullSignalsUrl| and false.
-1. Let |seller| be |auctionConfig|'s [=auction config/seller=].
-1. If |allTrustedScoringSignals| is null, then [=add a platform contribution=] with
-  [=trusted scoring signals failure bucket=], |realTimeContributionsMap|, and |seller|.
+1. If |allTrustedScoringSignals| is null, and |auctionConfig|'s

```

Comment:
I think the ...and reporting clause needs to be inside the conditional; this seems to inadvertently change when the "Otherwise..." on line 1959 runs

Reply:
Oh, never mind that, tha'ts qualified by "is an ordered map", so should be fine